packages/teko/src/Tpetra/Teko_BlockingTpetra.cpp

```diff
@@ -194,7 +194,8 @@ RCP<Tpetra::Vector<GO, LO, GO, NT> > getSubBlockColumnGIDs(
 // build a single subblock Epetra_CrsMatrix
 RCP<Tpetra::CrsMatrix<ST, LO, GO, NT> > buildSubBlock(
     int i, int j, const RCP<const Tpetra::CrsMatrix<ST, LO, GO, NT> >& A,
-    const std::vector<MapPair>& subMaps) {
+    const std::vector<MapPair>& subMaps,
+    const Teuchos::RCP<Tpetra::Vector<GO, LO, GO, NT> >& plocal2ContigGIDs) {
```

Comment:
Would it make sense to keep this backward compatible and use an optional argument?

Reply:
I don't think this is really user-facing, but it's easy enough to make it backward compatible by using an optional argument that I think that's a safer bet.